src/Layers/xrRenderPC_GL/r2_R_sun.cpp

```diff
@@ -214,8 +205,43 @@ Fvector3 wform(Fmatrix& m, Fvector3 const& v)
     r.w = v.x * m._14 + v.y * m._24 + v.z * m._34 + m._44;
     // VERIFY		(r.w>0.f);
     float invW = 1.0f / r.w;
-    Fvector3 r3 = {r.x * invW, r.y * invW, r.z * invW};
-    return r3;
+    return {r.x * invW, r.y * invW, r.z * invW};
+}
+
+Fvector3 wform(Fmatrix& m, glm::vec3 const& v)
```

Comment:
I don't see that `glm::mat4& m` is changed anywhere in this function. It should be made const reference then.